.github/workflows/support-files/notifications/send_message.js

```diff
@@ -89,7 +94,7 @@ async function sendKeybaseMessage(messageBody) {
     });
 
     const channel = {
-      name: 'nymtech_bot',
```

Comment:
I suggest you rather try the env var and fallback to the default

```suggestion
      name: context.env.KEYBASE_NYMBOT_TEAM || 'nymtech_bot',
```

Reply:
That's a better solution.